openedx/features/badging/views.py

```diff
@@ -32,7 +33,7 @@ def trophycase(request):
     return render_to_response(
```

Comment:
Let's remove this, this was added for QA to test json while showing template as well

`if request.GET.get('json'):
  return JsonResponse(trophycase_dict)
`

Reply:
Done